xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -114,12 +114,12 @@ class XAPIAN_VISIBILITY_DEFAULT Ranker : public Xapian::Internal::intrusive_base
      *                     It is initialised by DEFAULT set of Features by default.
      *                     Note: Make sure that this FeatureList object is the same as what was used during
      *                     preparation of the training file.
-     *  @exception FileNotFoundError will be thrown if file not found at supplied path
-     *  @return A vector of docids after ranking.
+     *  @exception FileNotFoundError will be thrown if file not found at supplied path.
+     *  @exception InvalidArgumentError will be thrown if the supplied MSet is empty.
```

Comment:
Drop: `@exception InvalidArgumentError will be thrown if the supplied MSet is empty.`